atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -52,7 +53,7 @@
     private static final String ROW_AND_COLUMN_NAME = "ts";
 
 
-    public static final String TIMESTAMP_TABLE = "_timestamp";
+    public static final String TIMESTAMP_TABLE = AtlasDbConstants.TIMESTAMP_TABLE;
```

Comment:
idk why this surrounding code doesn't just use AtlasDbConstants.TIMESTAMP_TABLE directly...